src/site/content/en/blog/samesite-cookies-explained/index.md

```diff
@@ -250,138 +268,163 @@ in a third-party context.
   </figcaption>
 </figure>
 
-{% Aside 'objective' %} If you provide a service that other sites consume such
-as widgets, embedded content, affiliate programmes, advertising, or sign-in
-across multiple sites then you should use `None` to ensure your intent is clear.
+{% Aside 'objective' %}
+
+If you provide a service that other sites consume such as widgets, embedded
+content, affiliate programmes, advertising, or sign-in across multiple sites
+then you should use `None` to ensure your intent is clear.
+
 {% endAside %}
 
 ## Changes to the default behavior without SameSite
 
-[Chrome 76](https://chromestatus.com/features/schedule) introduces a new
-`same-site-by-default-cookies` flag. Setting this flag will shift the default
-treatment of cookies to apply `SameSite=Lax` if no other `SameSite` value is
-provided. This is a move towards providing a more secure default and one that
-makes the intended purpose of cookies clearer to users. If you want to delve
-into the details, check out Mike West's
+While the `SameSite` attribute is widely supported, it has unfortunately not
+been widely adopted by developers. The open default of sending cookies
+everywhere means all use cases work but leaves the user vulnerable to CSRF and
```

Comment:
Spell out CSRF and include the link I provided above.

Reply:
Expanded and linked